.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: go
 
 go:
-  - 1.1
-  - 1.2
-  - 1.3
+  - 1.8.x
+  - 1.9.x
```

Comment:
Maybe you could also use `1.x` and change `tip` to `master`

https://docs.travis-ci.com/user/languages/go/#Specifying-a-Go-version-to-use